### DISCUSSION TITLE
What plugins/datapacks are used for /sethome?

### DISCUSSION THREAD

**Discussion by dark-swordsman** [GriefPrevention/GriefPrevention#880] [Q&A] [answered]
Hello,

I noticed in the config.yml that there is a command for `/sethome`. I tried googling and looking through the wiki for anything related to it. The Wiki and other google searches alluded to it being an available function somehow, but I couldn't find anything about `/sethome` functionality in GP.

Is there a specific plugin that works with Grief Prevention for this `/sethome` feature? 

My goal is to allow users to only `/sethome` in their own claims or claims they were given permission for. I have found other `/sethome` datapacks, but I am unsure if they actually work with GP correctly.

Thanks!

**Comment by Jikoo** [comment, marked as answer]
It's designed as a generic configurable catch-all, not for a specific plugin. That's just a convenient default for most users - similar to ender pearl blocking, you would expect players to not be able to teleport into your claim without permission. Be aware that GP doesn't actively seek out and block command aliases, so if you end up getting a plugin implementing that feature there may be additional aliases you need to add. For example, Essentials' `/sethome` has aliases `/esethome`, `/createhome`, and `/ecreatehome`, all of which would need to be listed.

I doubt a datapack would be supported, you're far better off getting a plugin to provide that functionality. Non-command-based home systems will not be compatible with this feature.

**Comment by dark-swordsman** [comment]
I see. Thanks for the help!
